packages/block-editor/src/components/iframe/index.js

```diff
@@ -139,6 +138,8 @@ function Iframe( {
 			const { documentElement } = contentDocument;
 			iFrameDocument = contentDocument;
 
+			documentElement.classList.add( '__iframe__html' );
```

Comment:
Let's name this in a way that is clear that this class is editor specific and not available on the frontend